### PR TITLE
start contributor faq

### PR DESCRIPTION
**Pull request by asmacdo**
I've heard this question a few times now, so I started an FAQ for contributors.

### CODE REVIEW

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

File: website/content/en/docs/contribution-guidelines/faq.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: FAQ
+weight: 200
+---
+
+#### Q: If find a problem, should I file a Pull Request or an Issue?
```
Comment (jmrodri) [inline website/content/en/docs/contribution-guidelines/faq.md:6]:
```suggestion
#### Q: If I find a problem, should I file a Pull Request or an Issue?
```

File: website/content/en/docs/contribution-guidelines/faq.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+A: We suggest that you start by filing an issue first because it can
+save you time if the change cannot be accepted for some reason.
+However, if it is easier to submit a PR than explain what you'd like to
```
Comment (jmrodri) [inline website/content/en/docs/contribution-guidelines/faq.md:10]:
```suggestion
However, if it is easier to submit a PR than to explain what you'd like to
```

File: website/content/en/docs/contribution-guidelines/faq.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: FAQ
+weight: 200
+---
+
+#### Q: If find a problem, should I file a Pull Request or an Issue?
+
+A: We suggest that you start by filing an issue first because it can
+save you time if the change cannot be accepted for some reason.
```
Comment (jmrodri) [inline website/content/en/docs/contribution-guidelines/faq.md:9]: I'm not sure I understand "save you time if the change cannot be accepted.." why would an issue save them time? Are you thinking that the time they save is the time used to implement the PR?

**Review by jmrodri — CHANGES_REQUESTED** [review]
A couple changes and a question that might result in some updates.

File: website/content/en/docs/contribution-guidelines/faq.md (hunk 3 above)
Comment (asmacdo) [reply website/content/en/docs/contribution-guidelines/faq.md:9]: The idea is that if they file an issue, and we say that we cant accept the change they won't waste time on implementation.

**Review by jmrodri — APPROVED** [review]

**Pull request merged**